src/main/java/core/basesyntax/stategy/handler/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.stategy.handler;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+    @Override
+    public void process(FruitTransaction transaction) {
+        Storage.fruits.put(transaction.getFruitName(), transaction.getQuantity());
```

Comment:
Direct access to the Storage map is not recommended. Consider encapsulating the access through a method that handles the logic of updating the storage. This will follow the encapsulation principle and make the code more maintainable.